packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -72,127 +71,126 @@ class AccountSignup extends React.Component {
+                                                has_error={!!(touched.password && errors.password)}
+                                                custom_feedback_messages={getErrorMessages().password_warnings}
+                                            >
+                                                <PasswordInput
+                                                    {...field}
+                                                    className='account-signup__password-field'
+                                                    label={localize('Create a password')}
+                                                    error={touched.password && errors.password}
+                                                    required
+                                                    value={values.password}
+                                                    onBlur={handleBlur}
+                                                    onChange={e => {
+                                                        const input = e.target;
+                                                        setFieldTouched('password', true);
+                                                        if (input) updatePassword(input.value);
+                                                        handleChange(e);
+                                                    }}
+                                                />
+                                            </PasswordMeter>
+                                        )}
+                                    </Field>
+                                    <Text as='p' size='xxs' className='account-signup__subtext'>
+                                        <Localize i18n_default_text='Strong passwords contain at least 8 characters, combine uppercase and lowercase letters, numbers, and symbols.' />
+                                    </Text>
+
+                                    <Button
+                                        className={classNames('account-signup__btn', {
+                                            'account-signup__btn--disabled':
+                                                !values.password || errors.password || isSubmitting,
+                                        })}
+                                        type='submit'
+                                        is_disabled={!values.password || !!errors.password || isSubmitting}
+                                        text={localize('Start trading')}
+                                        primary
+                                    />
+                                </div>
+                            )}
+                        </React.Fragment>
+                    </Form>
+                )}
+            </Formik>
+        </div>
+    );
+};
 
 AccountSignup.propTypes = {
+    enableApp: PropTypes.func,
     onSignup: PropTypes.func,
     residence_list: PropTypes.array,
+    isModalVisible: PropTypes.func,
```

Comment:
We may want to change this to `setModalVisible` moving forward.

Reply:
Is it confirmed?

Reply:
I think there is no need, we mostly use `set` prefix for setting state here, so it might create some confusion later on with developers thinking it's a state as well